### ISSUE TITLE
How do I change the background color of DatePicker?

### ISSUE THREAD

**Issue by mmgyce** [Promact/md2#74]

**Comment by ahirhasmukh** [comment]
@mmgyce You need to change css of datepicker(Custom css). Please check screenshot. I have tried to change color.

**Datepicker header color changed**
![datepicker](https://cloud.githubusercontent.com/assets/8941195/22550811/f168221c-e977-11e6-8f74-278b841be563.png)
 
**Inspect element css changes**
![css](https://cloud.githubusercontent.com/assets/8941195/22550819/f6129ca2-e977-11e6-9c78-a007247330e9.png)

**Comment by dharmeshpipariya-zz** [comment]
@mmgyce you can use our [SASS](https://github.com/Promact/md2/blob/master/src/lib/datepicker/datepicker.scss) to change the colors and theme too.

**Comment by jmpmscorp** [comment]
Hello guys and thank you for your work.

Could anyone help me? I'm becoming crazy trying to change default color of datepicker.

I have installed md2 through npm. I have changed default blue everywhere in 'datepicker.css'. I have changed $primary in 'datepicker.scss' but nothing happends, still default blue. I have added an overwrite class in my scss parent component (.md2-datepicker-header{ background: #00bfa5 }), still default blue. I have added the same class in my main style.scss but, again, nothing change.

I hope someone could help and guide me on how to change default color.

Regards.
